cmd/exchange_template/exchange_template_test.go

```diff
@@ -39,9 +41,8 @@ func TestCheckExchangeName(t *testing.T) {
 	}
 
 	for x := range tester {
```

Comment:
🐒 `tester` is not normal
Can also inline the test table
```suggestion
	for _, tt := range []struct{
		Name        string
		ErrExpected error
	}{
```